src/main/java/io/personium/client/ServiceCollection.java

```diff
@@ -76,14 +198,28 @@ public ServiceCollection(Accessor as, String path) {
     // */
     /**
      * This method configures a set of services.
-     * @param key Property Name
-     * @param value Property Value
+     * @param key name of service route
+     * @param value source of service route
      * @param subject Value of the service subject
      * @throws DaoException Exception thrown
      */
     public void configure(String key, String value, String subject) throws DaoException {
         RestAdapter rest = (RestAdapter) RestAdapterFactory.create(this.accessor);
```

Comment:
@yoh1496 Please remove this unused line.
```java
RestAdapter rest = (RestAdapter) RestAdapterFactory.create(this.accessor);
```

Reply:
I forgot to remove that. I modified.